flyteidl/protos/flyteidl/core/artifact_id.proto

```diff
@@ -17,8 +17,6 @@ message ArtifactKey {
 
 // Only valid for triggers
 message ArtifactBindingData {
-  uint32 index = 1;
```

Comment:
why was this removed? does this wire number need to be marked reserved?

Reply:
i don't want it anymore.